lib/demo.feature

```diff
@@ -55,6 +55,18 @@ Feature: demo features
 
     Then verify ".success-message" element has "Welcome" text
     Then verify ".fail-message" element has not "Welcome" text
+    Then verify ".success-message" element contains "Welcome" text
```

Comment:
`Then verify ".success-message" element has "Welcome" text` farki exact true/false olmasi midir? Eger oyleyse bunu step definition icerisinde belirtmek gerekir mi? Ikisi de bana ayni mesaji veriyor gibi geldi acikcasi.

Reply:
Farki exact true/false olmasıydı evet. Gereken testleri yaptım, dediğin gibi aynı mesajı veriyor.
page.should have_selector(selector, text: text, exact_text: false, wait: BddHelper.timeout)
yaparak aynı olmasının önüne geçebiliyoruz.